src/bindist/installdos

```diff
@@ -3,6 +3,8 @@
 SOURCE="$1"
 DRIVE_ROOT="$2"
 
+PROPRIETARY_DOS_INSTALLED_EXIT_CODE=100
```

Comment:
You mean installing to this location or installing
from this location?
Yes, I think for simplicity we should not support
installing _to_ that location and just leave the existing
install.c code for the freedos rescue stuff.
But I'd leave something to allow the user to
easily create the system-wide setup if he wants
to. It would be nice if the script can just support
some --dest, so that the user can use that and
then eg adjust $_hdimage or create symlinks
himself. We can even document that procedure
as it looks very simple, but we don't need to
implement it in a form of creating yet another
set of symlinks. That would be too much.